settings.json

```diff
@@ -644,6 +644,14 @@
       "requires": []
     }
   ],
+  "markdown": [
+    {
+      "command": "remark-language-server",
+      "requires": [
+        "npm"
+      ]
+    }
```

Comment:
```
"requires": [
  "npm",
  "typescript-language-server",
  "yaml-language-server"
]
```

Reply:
> I used `typescript-language-server` and `yaml-language-server` for reference.

With this I meant I looked how those are implemented to see how I should implement `remark-language-server`. Apart from that they’re not related / required.